UrlsSpiders/BBCUrlsSpider.py

```diff
@@ -61,11 +71,15 @@ def parseAll(self, response, year):
 				raise CloseSpider('LIMIT_URLS_COUNT')
 
 			return item
-	
-	def parseNews(self, response):
-		year = self.getYearFromArray(response.xpath("//@data-seconds"))
 
-		return self.parseAll(response, year)
+	def cleanUrl(self, url):
```

Comment:
Why use clean if the match can be just with http and without the pal.live?

Reply:
You are right about pal.live part, it's actually redandent, but the function is still needed for http/https.

Reply:
you can add http to the regex

Reply:
Yes, I can, but I don't can to be sure how a URL will be parsed (as http or https) and I can't just to filter out one of them.

Reply:
OK